core/templates/pages/blog-dashboard-page/blog-dashboard-page-root.component.html

```diff
@@ -0,0 +1,105 @@
+<oppia-angular-root>
+  <oppia-base-content>
+    <navbar-pre-logo-action>
+      <oppia-blog-post-editor-pre-logo-action>
+      </oppia-blog-post-editor-pre-logo-action>
+    </navbar-pre-logo-action>
+    <navbar-breadcrumb>
+      <oppia-blog-dashboard-navbar-breadcrumb>
+      </oppia-blog-dashboard-navbar-breadcrumb>
+    </navbar-breadcrumb>
+    <content>
+      <oppia-blog-dashboard-page>
+      </oppia-blog-dashboard-page>
+    </content>
+    <page-footer>
+    </page-footer>
+  </oppia-base-content>
+</oppia-angular-root>
+<style>
+  button:focus {
+        border: unset;
+        outline: unset;
+      }
+
+      @media(max-width: 768px) {
+        .collapse:not(.show) {
+          display: flex;
+        }
+
+        .navbar-brand {
+          display: none;
+        }
+
+        .oppia-navbar-brand-name.d-none {
+          /* !important is necessary otherwise d-none class of base components is not overwritten(!important
+          is being used there). */
+          display: flex !important;
+          margin-left: -10px;
+        }
+
+        .ng-cloak {
+          display: flex !important;
+          flex-direction: row;
+        }
+
+        .float-left {
+          margin-left: 5%;
+        }
+      }
+      @media (max-width: 500px) {
+        .float-left {
+          margin-left: 2%;
+        }
+      }
+
+  @media(max-width: 870px) {
+    .blog-dashboard-card {
+      margin: 20px;
+      width: unset;
+    }
+
+    .new-post-button {
+      margin-bottom: unset;
+    }
+
+    :host /deep/ .mat-tab-label {
+      font-family: 'Roboto', Arial, sans-serif;
+      font-size: 1.1rem;
+    }
+  }
+
+  @media(max-width: 620px) {
+    .blog-dashboard-card-title {
+      color: #333;
+      font-family: 'Roboto', Arial, sans-serif;
+      font-size: 1.6rem;
+      font-weight: 400;
+      text-align: left;
+    }
+
+    .new-post-button {
+      font-size: 14px;
+      margin-bottom: unset;
+      padding-left: 0;
+      padding-right: 0;
+      width: 96px;
+    }
+
+    .blog-author-bio-container {
+      font-size: 14px;
+      line-height: 16px;
+    }
+
+    :host /deep/ .mat-card-header-text {
+      margin-right: 0;
+    }
```

Comment:
Why do we need `:host` here?

Reply:
some of these changes were automatic, when i pulled the latest git. will  update these

Reply:
Is it possible to check the steps you followed last time to update the branch? (I wanted to make sure it's correct, or else it will continuously create problems for you!)

Also, I don't see such CSS in any branch, so I'm worried it's getting added automatically by any extensions (like copilot or something like that).